roles/importer/files/importer/common.py

```diff
@@ -81,7 +81,9 @@ def __str__(self):
 """Used for storing state during import process per management"""
 class ImportState():
     ErrorCount: int
-    ChangeCount: int
+    # ChangeCount: int
```

Comment:
Leaving this in as commented out might be confusing in the future. I think it's generally better to delete obsolete code - we can go back in the git history if needed.